yarn-project/stdlib/src/logs/tx_scoped_l2_log.ts

```diff
@@ -88,6 +95,7 @@ export class TxScopedL2Log {
       this.dataStartIndexForTx === other.dataStartIndexForTx &&
       this.logIndexInTx === other.logIndexInTx &&
       this.blockNumber === other.blockNumber &&
+      this.blockHash === other.blockHash &&
```

Comment:
```suggestion
      this.blockHash.equals(other.blockHash) &&
```
We need to set up a linter rule to catch these kind of errors, they happen way too often. That, or convert L2BlockHash into a branded type over a string.

Reply:
Oof, nice catch